### PR TITLE
Set lts version as default node when set lts param [semver:patch]

### LINKED ISSUE #82: node/install options "lts: true" not set the lts version as default
## What happened:

I add node/install like this:

```
- node/install:
       install-yarn: true
       install-npm: false
       lts: true
```

But, the lts version not set as default.

This is the example evidence: https://app.circleci.com/pipelines/github/berv-uni-project/tweety-ui/34/workflows/f5060f49-1048-47d3-b6d4-8d73b897278f/jobs/34

## Expected behavior:

The lts version will be default.

### PR DESCRIPTION
**Pull request by berviantoleo**
Fix #82

### CODE REVIEW

File: src/scripts/install-nvm.sh [hunk 1]
```diff
@@ -10,6 +10,8 @@ fi
 
 if [ "$NODE_PARAM_LTS" = "1" ]; then
     nvm install --lts
+    nvm alias default lts
+    nvm use --lts
```
Comment (Kurt-von-Laven) [inline src/scripts/install-nvm.sh:14]: It seems like this could be simplified by removing this line, and moving line 18 outside of the conditional?

**Review by Kurt-von-Laven — COMMENTED** [review]
I am not a maintainer, but I am guessing the rationale for appending the use command to `$BASH_ENV` is that each `run` command is run in a separate shell on CircleCI.

File: src/scripts/install-nvm.sh (hunk 1 above)
Comment (berviantoleo) [reply src/scripts/install-nvm.sh]: Thanks for the recommendation, how about now?

Comment (Kurt-von-Laven) [reply src/scripts/install-nvm.sh]: Looks good to me!

**Review by Kurt-von-Laven — APPROVED** [review]
I believe this change is correct now.

**Review by KyleTryon — APPROVED** [review]
@berviantoleo Thank you for your addition and @Kurt-von-Laven thank you again for your review!

**Review by felicianotech — APPROVED** [review]

**Comment by berviantoleo** [comment]
Seems I broke some test. I am so sorry about that. I need to recheck how to solve those test.

**Comment by KyleTryon** [comment]
It appears that `lts` itself is not a valid alias.

```
Computing checksum with shasum -a 256
Checksums matched!
Now using node v14.17.1 (npm v6.14.13)
Creating default alias: default -> lts/* (-> v14.17.1 *)
! WARNING: Version 'lts' does not exist.
default -> lts (-> N/A)
```
https://app.circleci.com/pipelines/github/CircleCI-Public/node-orb/754/workflows/8892bcf5-d5b3-40b9-9157-92d9ff64e8b5

perhaps we need to use `lts/*`

**Review by KyleTryon — CHANGES_REQUESTED** [review]
Tests failed: https://app.circleci.com/pipelines/github/CircleCI-Public/node-orb/754/workflows/8892bcf5-d5b3-40b9-9157-92d9ff64e8b5

**Review by KyleTryon — APPROVED** [review]

**Comment by KyleTryon** [comment]
Merged, thank you @berviantoleo

**Pull request merged**